scan.md

```diff
@@ -0,0 +1,38 @@
+# SCAN
+
+Documentation on the Seattle Coronavirus Assessment Network (SCAN) project
+
+## Adding a new language
+
+Checklist
+1. Insert `<span>` tags into the REDCap project fields.
+   - Download the existing data dictionary with the [download-data-dictionary] script.
+   - Upload a modified data dictionary with injected HTML attributes via [upload-data-dictionary].
+2. Enable REDCap DETs for the new project.
+3. Update the REDCap DET ETL with the new language's project ID and ISO code (see [example]).
```

Comment:
Need to add a new cronjob for the new REDCap DET ETL

Reply:
Added!